spec.bs

```diff
@@ -579,46 +580,77 @@ An auction config is a [=struct=] with the following items:
 <dl dfn-for="auction config">
 : <dfn>seller</dfn>
 :: An [=url/origin=].
+  The origin of the seller running the ad auction.
 : <dfn>decision logic url</dfn>
 :: A [=URL=].
+  The URL to fetch the seller's JavaScript from.
 : <dfn>trusted scoring signals url</dfn>
 :: Null or a [=URL=].
+  Provide a mechanism for making real-time data available for use at scoring time.
 : <dfn>interest group buyers</dfn>
 :: Null or a [=list=] of [=url/origin=].
+  Owners of interest groups allowed to participate in the auction.
 : <dfn>auction signals</dfn>
 :: Null or a [=string=].
+  Opaque JSON data passed to both sellers' and buyers' script runners.
 : <dfn>seller signals</dfn>
 :: Null or a [=string=].
+  Opaque JSON data passed to the seller's script runner.
 : <dfn>seller timeout</dfn>
-:: A duration. Defaulting to 50.
+:: A duration. Defaulting to 50 milliseconds.
+  Restricts the runtime of the seller's scoreAd() script.

```

Comment:
You should eventually link this to the definition of the `scoreAd()` function, but for now, monospace is good enough:

```suggestion
  Restricts the runtime of the seller's `scoreAd()` script.
```